eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Binary/RoundTrip.hs

```diff
@@ -62,7 +62,7 @@ roundTripConwayEraTypesSpec = do
   describe "Conway State Types" $ do
     roundTripShareEraTypeSpec @era @EnactState
     roundTripShareEraTypeSpec @era @GovActionState
-    roundTripShareEraTypeSpec @era @Proposals
+    -- roundTripShareEraTypeSpec @era @Proposals -- TODO: @aniketd
```

Comment:
```suggestion
    pendingWith "TODO: @aniketd" -- roundTripShareEraTypeSpec @era @Proposals
```

Reply:
This has been retored in the subsequent PR :+1: